components/Board/Profile.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+export default function Profile(props) {
```

Comment:
Only use arrow functions for components. `rafce` gives you an arrow function. Keeps code consistent with other files. There are situations where arrow functions work and regular functions do not work. Javascript is also moving towards arrow functions as well due to some of the various advantages it has over regularly defined functions.